core/server/proxy/src/main/java/alluxio/proxy/s3/CompleteMultipartUploadHandler.java

```diff
@@ -87,20 +85,14 @@ public void handle(String s, Request request, HttpServletRequest httpServletRequ
       if (!s.startsWith(mS3Prefix)) {
         return;
       }
-      // Build log message capturing the request details
-      // TODO(czhu): Move this to a separate logging handler
-      StringBuilder sb = new StringBuilder();
-      sb.append("Alluxio S3 API received ");
-      sb.append(request.getMethod());
-      sb.append(" request: URI=");
-      sb.append(s);
-      if (request.getQueryString() != null) { sb.append("?").append(request.getQueryString()); }
-      sb.append(" User=");
+      if (!request.getMethod().equals("POST") || request.getParameter("uploadId") == null) {
+        return;
+      } // Otherwise, handle CompleteMultipartUpload
       final String user;
       try {
         // TODO(czhu): support S3RestServiceHandler.getUserFromSignature()
         //             Ideally migrate both to S3RestUtils and make them static
-        user = S3RestServiceHandler.getUserFromAuthorization(
+        user = S3RestUtils.getUserFromAuthorization(
```

Comment:
Does `CompleteMultipartUploadHandler` not see the filtered version of the "Authorization" header from `S3AuthenticationFilter`? If it doesn't, then I'd prefer to remove that behaviour from `S3AuthenticationFilter`.

Reply:
Compared with other handler and filter, the `CompleteMultipartUploadHandler` use a totally different handler way. they are even not the same lib. That's why it executes before all handlers and filters. I plan to refactor this handler late, but now he can only be like this.